crates/subspace-networking/src/node.rs

```diff
@@ -338,7 +343,8 @@ impl Node {
             })
             .await?;
 
-        Ok(result_receiver.await?)
+        // TODO: A wrapper that'll immediately cancel query on drop
+        Ok(result_receiver)
```

Comment:
Shouldn't we conceal the streaming complexity here? We could implement `Quorum::One` policy here by awaiting the first result.

Reply:
I think what we'll do instead is we'll expose quorum as an argument on `put_value` instead.